geopandas/io/tests/test_arrow.py

```diff
@@ -531,3 +532,12 @@ def test_non_fsspec_url_with_storage_options_raises():
     with pytest.raises(ValueError, match="storage_options"):
         test_dataset = "naturalearth_lowres"
         read_parquet(get_path(test_dataset), storage_options={"foo": "bar"})
+
+
+@pytest.mark.skipif(
+    pyarrow.__version__ < LooseVersion("5.0.0"),
+    reason="pyarrow.fs requires pyarrow>=5.0.0",
```

Comment:
Is this skip needed? (it's certainly available before 5.0, but maybe not on the oldest version that we are testing, that's something I would need to check)

Reply:
This is intended to match the check at https://github.com/geopandas/geopandas/pull/2107/files#diff-d76448b39577164a1737b12aa472f6bac6e2fe20a3e1362aab16eda8ab1c0283R362. Should one of these versions be changed?

Reply:
Ah, yes, I put that there ;) 
I think that was because we only started to do prebuffering/coalescing with S3 with that version, so with older arrow there is certainly no benefit in passing the arrow filesystem.